workflows/aspace_harvest.py

```diff
@@ -82,12 +82,10 @@ def check_if_records(**context):
                      task_definition=f"airflow-{air_env}-mario",
                      overrides={'containerOverrides': [{
                                   'command': [f"--url={es_url}",
-                                              '--index=aspace_2019_12',
                                               'ingest',
-                                              '--prefix=aspace',
-                                              '--type=archives',
+                                              '--index=aspace_2019_12',
```

Comment:
We'll need to change this index to use `-` instead of `_` based on the discussion we just had about the open Mario PR. (Side note: this is one place where we sort of do need to maintain the ability to create a specifically named index or come back and futz with this script whenever we need to change it)

Reply:
We discussed on Slack and Helen proposed a better alternative which is to not provide an index at all, which will create one if there is none and use the current production index if there is.

Reply:
This is now updated, we'll still need to manually redo the full ingest in order for this to work but that will be a one-time deal.